File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] arr = s.split(" ");
        String min = arr[0];
        String max = arr[0];
        for(int i = 1; i < arr.length; i++){
            if(Integer.parseInt(arr[i]) < Integer.parseInt(min)){
                min = arr[i];
            }
            if(Integer.parseInt(arr[i]) > Integer.parseInt(max)){
                max = arr[i];
            }
        }
        answer = min + " " + max;
        return answer;
    }
}

File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    s = s.split(" ")
    min = 9999999
    max = -9999999
    for i in s:
        if(min > int(i)): min = int(i)
        if(max < int(i)): max = int(i)
    answer = str(min) + ' ' + str(max) 
    return answer